fix play rejection description (#21921)
mdn/content: files/en-us/web/api/htmlmediaelement/play/index.md

### Exceptions

> The promise's **rejection handler** is called with an exception {{domxref("DOMException")}}

Reviewer: ```suggestion
The promise's **rejection handler** is called with a {{domxref("DOMException")}} object
```
